### PR TITLE
integ tests: Create and remove_linux_bridge

### PR DESCRIPTION
**Pull request by SvetaHBar**

### CODE REVIEW

File: tests/integration/linux_bridge_test.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+#
+# Copyright 2018 Red Hat, Inc.
```
Comment (EdDev) [inline tests/integration/linux_bridge_test.py]: 2019

File: tests/integration/linux_bridge_test.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+          priority: 32768
+      port:
+        - name: eth1
+          stp-hairpin-mode: false
+          stp-path-cost: 100
+          stp-priority: 32
+"""
+
+
+def test_remove_linux_bridge_with_minimum_desired_state(eth1_up):
```
[the remark this answers is not in the document]
Comment (SvetaHBar) [reply tests/integration/linux_bridge_test.py]: Done

**Comment by tyll** [comment]
@SvetaHBar please rebase the commit to resolve the conflict.

**Comment by SvetaHBar** [comment]
Looks like this PR commit has been already merged through #252 .

**Pull request closed without merging**